src/pages/GamePage/GameWord/WordCell.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactNode } from 'react';
+
+const positions = ['center', 'left', 'right'];
+const margins = ['mt-auto', 'mb-auto'];
+const WordCell = ({
```

Comment:
```suggestion
const positions = ['center', 'left', 'right'];
const margins = ['mt-auto', 'mb-auto'];

const WordCell = ({
```
변수와 컴포넌트의 구분이 현재도 잘 되지만 이렇게 변경한다면 좀 더 가독성이 좋아질 것 같아요! 👍